azext_iot/central/commands_monitor.py

```diff
@@ -15,7 +15,7 @@
     CentralHandlerArguments,
     TelemetryArguments,
 )
-from azext_iot.monitor.property import start_property_monitor
+from azext_iot.monitor.property import PropertyMonitor
```

Comment:
Action: do NOT import anything from `monitor` namespace until there has been a call to `init_monitoring`

Details:

Since this file is importing stuff from `monitor/` folder, we have to be really careful about users without uamqp. 

Even if your code doesn't use UAMQP, the way imports are done, theres a valid risk that we could break users without uamqp

To do this, we have to do an `init_monitor` call.

So we have 2 paths:

1) init_monitor call before importing anything from `.monitor`. This function gets eventually called as part of instantiating `TelemetryArguments` object.
2) Move this code to `central/` folder to avoid importing from monitor at all (not preferred)

I'd like to eventually delete all references to monitor in the commands_monitor.py class, since our dependency on uamqp makes us more fragile

Reply:
verified that there is no dependency on uamqp with current flow.